pom.xml

```diff
@@ -74,6 +74,7 @@
         <hibernate.version>6.3.1.Final</hibernate.version>
         <spring.version>6.2.1</spring.version>
         <liquibase.version>4.30.0</liquibase.version>
+        <lombok.version>1.18.36</lombok.version>
```

Comment:
This is defined in the parent pom and should be removed here.

```suggestion
```

Reply:
Done